scripts/ngless-cwl/cwl/ngless-count.cwl

```diff
@@ -6,49 +6,48 @@
 cwlVersion: "cwl:v1.0"
```

Comment:
That is a funny quirk, `cwlVersion: v1.0` is sufficient. I just made a PR to `argparse2tool` to fix that there: https://github.com/erasche/argparse2tool/pull/43